cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,14 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden.
+    if xblock.category == 'sequential' and getattr(parent_xblock, "is_entrance_exam", False):
```

Comment:
according to me, parent_xblock should not be none in case of entrance exam when xblock category is sequential' . entrance_exam.py creating entrance exam and its subsection.